test/net/sourceforge/kolmafia/request/RelayRequestTest.java

```diff
@@ -458,6 +459,90 @@ public void handlesOverloadedFunction(String type, String identifier) {
       }
     }
 
+    @ParameterizedTest
+    @CsvSource({
+      "Bounty,none,",
+      "Bounty,bean-shaped rock,",
+      "Class,none,-1",
+      "Class,Seal Clubber,1",
+      "Coinmaster,none,",
+      "Coinmaster,The Black Market,",
+      "Effect,none,-1",
+      "Effect,Confused,3",
+      "Element,none,",
+      "Element,cold,",
+      "Familiar,none,-1",
+      "Familiar,Mosquito,1",
+      "Item,none,-1",
+      "Item,seal-clubbing club,1",
+      "Location,none,0",
+      "Location,The Sleazy Back Alley,112",
+      "Modifier,none,",
+      "Modifier,Item Drop,",
+      "Monster,none,0",
+      "Monster,spooky vampire,1",
+      "Path,none,-1",
+      "Path,Standard,22",
+      "Phylum,none,",
+      "Phylum,construct,",
+      "Servant,none,",
+      "Servant,Assassin,",
+      "Skill,none,-1",
+      "Skill,Liver of Steel,1",
+      "Slot,none,",
+      "Slot,hat,",
+      "Stat,none,",
+      "Stat,Muscle,",
+      "Thrall,none,",
+      "Thrall,Lasagmbie,",
+      "Vykea,none,",
+      "Vykea,level 1 couch,",
+    })
+    public void identityReturnsSameIdentifier(
+        String type, String identifierString, String identifierNumberString) {
```

Comment:
```suggestion
        String type, String identifierString, Integer identifierNumber) {
```
(and a corresponding change further down)

just shifts the integer parsing into the test framework